website/docs/r/storage_mover_agent.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Storage Mover"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_mover_agent"
+description: |-
+  Manages a Storage Mover Agent.
+---
+
+# azurerm_storage_mover_agent
+
+Manages a Storage Mover Agents.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "East US"
+}
+
+resource "azurerm_storage_mover" "example" {
+  name                = "example-ssm"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_storage_mover_agent" "example" {
+  name             = "example-sa"
+  storage_mover_id = azurerm_storage_mover.example.id
+  arc_resource_id  = "${azurerm_resource_group.example.id}/providers/Microsoft.HybridCompute/machines/examples-hybridComputeName"
+  arc_vm_uuid      = "3bb2c024-eba9-4d18-9e7a-1d772fcc5fe9"
+  description      = "Example Agent Description"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Storage Mover Agent. Changing this forces a new resource to be created.
+
+* `arc_resource_id` - (Required) Specifies the fully qualified resource ID of the Hybrid Compute resource for the Storage Mover Agent. Changing this forces a new resource to be created.
+
+* `arc_vm_uuid` - (Required) Specifies the VM UUID of the Hybrid Compute resource for the Storage Mover Agent. Changing this forces a new resource to be created.
```

Comment:
why is this needed, we provide the Resource ID?

Reply:
Maybe the description should be updated as below, refer [here ](https://github.com/Azure/azure-rest-api-specs/blob/5042eea1a5a0471073e37d874687aad62f6cffbb/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-11-10/HybridCompute.json#L1293)and [here](https://learn.microsoft.com/en-us/windows-hardware/drivers/bringup/smbios).
![image](https://user-images.githubusercontent.com/39109137/231925265-c24d62f3-bd0e-4ef2-afc0-e28ca3e19e6a.png)